core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -389,28 +406,26 @@
                 0.8, ConfigDef.Importance.LOW,
                 KAFKASTORE_SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR_DOC)
         .define(KAFKASTORE_GROUP_ID_CONFIG, ConfigDef.Type.STRING, "",
-                ConfigDef.Importance.LOW, KAFKASTORE_GROUP_ID_DOC);
+                ConfigDef.Importance.LOW, KAFKASTORE_GROUP_ID_DOC)
+        .define(SCHEMAREGISTRY_RESOURCE_EXTENSION_CONFIG, ConfigDef.Type.STRING, "",
```

Comment:
currently I don't really see a need for supporting multiple extensions for the reason that you could simply add multiple filters from one registered extension.